lib/money/money.rb

```diff
@@ -25,6 +25,9 @@ class Money
   # Raised when smallest denomination of a currency is not defined
   class UndefinedSmallestDenomination < StandardError; end
 
+  # Raised when Money is intialised without currency and no default currency
+  class DefaultCurrencyNotSetOrPassed < StandardError; end
```

Comment:
nit: Probably `DefaultCurrencyNotSet` is enough

Reply:
Ok. I will change it.